Makefile.versions

```diff
@@ -1,10 +1,10 @@
 #
 # CRIU version.
 CRIU_VERSION_MAJOR	:= 3
-CRIU_VERSION_MINOR	:= 15
+CRIU_VERSION_MINOR	:= 16
 CRIU_VERSION_SUBLEVEL	:=
 CRIU_VERSION_EXTRA	:=
-CRIU_VERSION_NAME	:= Titanium Falcon
+CRIU_VERSION_NAME	:= Petrified Puffin (TODO)
```

Comment:
Is `(TODO)` in the version name intentional?

Reply:
No. Removed.